docs/config/io_helidon_tracing_jaeger_JaegerTracerBuilder.adoc

```diff
@@ -1,6 +1,6 @@
 ///////////////////////////////////////////////////////////////////////////////
 
-    Copyright (c) 2022 Oracle and/or its affiliates.
+    Copyright (c) 2023 Oracle and/or its affiliates.
```

Comment:
Do not change this file (or any in the `docs/config` directory) manually.

They are all generated.